src/python/strelka/scanners/scan_pdf.py

```diff
@@ -117,7 +119,9 @@ def scan(self, data, file, options, expire_at):
                             )
                         self.files.append(extract_file)
                         self.flags.append("extracted_text")
-                except:
+                except Exception:
+                    traceback.print_exc()
```

Comment:
This is fairly standard practice for what we want. Catch the exception, log it, and continue. Completely hiding it makes errors impossible to hunt down